src/app/components/Detail/WorkDetail.jsx

```diff
@@ -140,6 +140,10 @@ class WorkDetail extends React.Component {
     } else if (query.showAll) {
       query.showAll = 'true';
       this.setState({ shouldShowAll: true });
+    } else {
+      // No showAll in query
+      query.showAll = 'false';
+      this.setState({ shouldShowAll: false });
```

Comment:
should `false` be a string here? seems inconsistent with the change above

Reply:
It should be a boolean - query.showAll is a string because that's is expression in the query url string, but it's much simpler to set the state to be boolean values.  

Or, put in a robust "parseQueryToString" and "parseStringToQuery", but since showAll is the only thing that would need to be transformed, it might be overkill.

Reply:
Oh also, this is what fixes the subject bug - before, if you didn't pass a showAll into query, it would error.  The subject links don't pass showAll into query (they could, but this is still a case that should be handled)